### PR TITLE
make ansible playbook run in local istio version

### PR DESCRIPTION
**Pull request by LillyWu**
To issue: https://github.com/istio/istio/issues/7250
To improve PR: https://github.com/istio/istio/pull/7268

### CODE REVIEW

**Comment by LillyWu** [comment]
@geoand  this is the feature that I promise to implement last week. (enable ansible playbook install istio with local release)

@geoand  @sdake  as the last PR https://github.com/istio/istio/pull/7268 has been merged, I created a new feature request issue:  https://github.com/istio/istio/issues/7355 to sort out all the things we could do to make ansible installation better

File: install/kubernetes/ansible/istio/tasks/add_to_path.yml [hunk 1]
```diff
@@ -1,3 +1,7 @@
+- name: Set istio dir
+  set_fact:
+    istio_dir: "{{ istio_k8s_dir | dirname | dirname }}"
```
Comment (geoand) [inline install/kubernetes/ansible/istio/tasks/add_to_path.yml]: Shouldn't this be `{{ istio_k8s_dir | dirname }}` or is there a specific need for the second `dirname` ?

Comment (LillyWu) [reply install/kubernetes/ansible/istio/tasks/add_to_path.yml]: @geoand  yes. `istio_k8s_dir` is  `/home/lilly/istio/install/kubernetes` and ISTIO_BIN_DIR is under `/home/lilly/istio/bin`, so it needs to get the parent of the parent of `istio_k8s_dir`.

**Pull request merged**